wp-redis.php

```diff
@@ -73,7 +81,7 @@ function wp_redis_get_info() {
 	} else {
 		$uptime_in_days .= ' days';
 	}
-	$database  = ! empty( $redis_server['database'] ) ? $redis_server['database'] : 0;
+	$database  = ! empty( $redis_server['database'] ) ? $redis_server['database'] : $database;
```

Comment:
I know this is not your code @timnolte, but raising the same thing I mentioned to Chris earlier on another ternary— with this else case, we're evaluating $database = $database. Chris thinks I just hate ternaries.
```suggestion
	if ( ! empty( $redis_server['database'] ) {
		$database = $redis_server['database'];
	}
```